boards/arm/twr_ke18f/twr_ke18f.dts

```diff
@@ -132,13 +132,6 @@
 			zephyr,code = <INPUT_KEY_1>;
 		};
 	};
-
-	flexio: flexio@4005a000 {
```

Comment:
I think we should mark the new (SOC level) flexio node as `status=okay` at the board level for the TWR-KE18F